.github/workflows/continuous-integration-dotnet.yml

```diff
@@ -0,0 +1,80 @@
+name: SonarCloud scanning and coverage generation
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: [ master ]
+    types: [ opened, synchronize, reopened ]
+
+env:
+  JAVA_VERSION: '11'
+
+jobs:
+  build-and-test:
+    runs-on: windows-latest
+    steps:
+    
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.ref }}
+        fetch-depth: 0 # Shallow clones disabled for a better relevancy of SC analysis
+
+    - name: Install Nuget
+      uses: nuget/setup-nuget@v1
+      with:
+        nuget-version: ${{ needs.set-env.outputs.nuget_version }}
+
+    - name: Add msbuild to PATH
+      uses: microsoft/setup-msbuild@v1.1
+
+    - name: NuGet restore
+      run: nuget restore DFE.SIP.API.SharePointOnline.sln
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v3
+      with:
+        java-version: ${{ env.JAVA_VERSION }}
+        distribution: 'zulu'
```

Comment:
Is there an advantage to using `zulu` instead of `microsoft` here? https://github.com/actions/setup-java#supported-distributions

Reply:
Zulu is the one that was in the example config from SonarCloud. I don't know if there's any significance behind it. Happy to try changing it to the microsoft one?